velox/exec/Driver.cpp

```diff
@@ -1090,6 +981,72 @@ folly::dynamic Driver::toJson() const {
   return obj;
 }
 
+template <typename Func>
+auto Driver::withDeltaCpuWallTimer(
```

Comment:
```
s/withDeltaCpuWallTimer/runOpWithDeltaCpuWallTimer/?
```

Reply:
Is it ok to keep this name? I actually think `withXXX` is more conformant to the industry coding conventions.

Reply:
Either way is fine but this seems to be just a wrapper to run an operator method with time recording?